src/components/Home.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
+import { Button } from 'react-bootstrap';
 
 const Home = () => {
   return (
-    <div>
-      <h1> Hello This is Home page</h1>
-    </div>
+    <main className="text-center">
+      <section>
+        <h2>{"{ CYF Quiz }"}</h2>
+        <p>Hello Coder !<br />
+        Are you ready for the next challenge ?<br />
+        Take a quiz !</p>
```

Comment:
For lines 9-11, I was thinking it would be one paragraph with no line breaks (I know that the screenshot has line breaks). However, if you want to have line breaks then you should use 3 `p`s with no `br`. In web development, we try not to use `br` anymore since you should always be able to do spacing with CSS. Also, punctuation marks don't have spaces before them, so there would not be a space before `!` and `?`

Reply:
I see, thanks for this feedback :) , this is new to me.
yes, I will update that.